pages/api/circulating-supply.ts

```diff
@@ -31,6 +31,8 @@ const exa = {
 } as const;
 
 export default async function (req: NextApiRequest, res: NextApiResponse) {
+  res.setHeader('Cache-Control', 's-maxage=86400');
+  console.log('CACHE MISS');
```

Comment:
this only enables cache in the client. we need caching on the server side.

Reply:
as discussed, this is true, but in this case the "client" is an API gateway that handles caching server side.